src/models/userModel.ts

```diff
@@ -25,5 +25,18 @@ class User {
     const users = await db(this.table).select().returning('*');
```

Comment:
I think this part will return all the users along with their hashed password? Is that is the case, we would not want any user to view others' password in their response. IMPORTANT that u address it.

Reply:
One user will not get other user's passwordHash as response, as I would delete getAllUsers route later (it is easy to use route without any body for authentication check, so using it for testing). But still I will try to not return passwordHash as response whenever possible. Thanks for advice.